packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -1000,7 +1037,7 @@ export async function upgradeGenericHelmChart(
   } else {
     console.info(`Upgrading helm release ${releaseName}`)
     await helmCommand(
-      `helm upgrade --install ${valuesOverride} ${releaseName} ${chartDir} --namespace ${namespace} ${parameters.join(
+      `helm upgrade --install ${valuesOverride} ${releaseName} ${chartDir} --timeout 120h --namespace ${namespace} ${parameters.join(
```

Comment:
Any reason for having such a high timeout here? Could any lower value be enough?

Reply:
Based on my experience, we've had very long-running migrations in the past and since migrations run in a hook now, we have to be conservative here.